deploy/k8s/v0.3.0/overlays/ecr/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base
+images:
+- name: amazon/aws-efs-csi-driver
+  newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-efs-csi-driver
+  newTag: v0.3.0
+- name: quay.io/k8scsi/livenessprobe
+  newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/csi-liveness-probe
```

Comment:
From 2.0.0 onward, the name is going to match the upstream name. i.e. it will be 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/livenessprobe:v2.0.0. I am not sure how to capture this so we don't forget to change it...

Reply:
i.e.: livenessprobe as opposed to csi-liveness-probe

Reply:
If we don't change name, it the image pull will fail since the old image name with v2 won't exist. We should test we are able to pull from dockerhub/ECR locally before cutting the next release.